ibridges/rules.py

```diff
@@ -25,21 +24,31 @@ def execute_rule(session: Session, rule_file: str, params: dict,
         The irods session
     rule_file : str
         Name of the iRODS rule file, or a file-like object representing it.
-    params : dict
-        Rule arguments.
-    output : str
-        Rule output variable(s).
+    kwargs : dict example keywords:
+        body    : str,rulename
+            Name of the rule on the irods server.
+        params : dict
+            Rule arguments.
+        instance_name : str
+            changes between irods rule language and python rules.
+        output : str
+            Rule output variable(s).
+
 
     Returns
     -------
     tuple
         (stdout, stderr)
 
     """
```

Comment:
It would also be nice if you have some examples to add them to the docstring.

Reply:
For as far as I can see you are using the quickstart and other tutorials for this?

Reply:
Yes, you're right that currently we're using mostly the quickstart/tutorials for that. I think in this case rules is not something that first time users will have to worry about, which is why I suggested putting it in the docstring/docs on readthedocs. It's also okay the way it is now, since our docs (and especially the API part) need some love at some point and that can be done at that point. I'll just have to remember to check this PR.